src/readme.spec.js

```diff
@@ -90,6 +90,7 @@ describe('readme', () => {
         'https://github.com/kefranabg/readme-md-generator#readme',
       projectHomepage:
         'https://github.com/kefranabg/readme-md-generator#readme',
+      projectDemoUrl: 'kefranabg',
```

Comment:
You can use the url of this repos as an example

Reply:
I have changed the `projectDemoUrl ` to this repos url